src/plugins/embeddable/public/lib/panel/embeddable_panel.test.tsx

```diff
@@ -213,9 +213,9 @@ const renderInEditModeAndOpenContextMenu = async (
 };
 
 test('HelloWorldContainer in edit mode hides disabledActions', async () => {
-  const action = {
+  const action: Action = {
     id: 'FOO',
-    type: 'FOO',
+    type: 'FOO' as ActionType,
```

Comment:
If you don't want to do the `declare module` thing, like in tests, this casting `as ActionType` works.